addons/test_mail/tests/test_performance.py

```diff
@@ -967,7 +967,7 @@ def test_complex_tracking_template(self):
         rec1 = rec.with_context(active_test=False)      # to see inactive records
         self.assertEqual(rec1.message_partner_ids, self.partners | self.env.user.partner_id | self.user_portal.partner_id)
 
-        with self.assertQueryCount(__system__=20, emp=21):
+        with self.assertQueryCount(__system__=20, employee=21):
```

Comment:
why this change ?

Reply:
That was an error (user is employee and not emp). And without that change, I get a query count error.